workflow/snakemake_rules/common.smk

```diff
@@ -223,6 +223,13 @@ def _get_upload_inputs(wildcards):
 
     origin = config["S3_DST_ORIGINS"][0]
 
+    # the 100k sample is unique in that we only want to upload 2 files. Note that the S3_DST_ORIGINS will specify the directory
+    if len(config["builds"].keys())==1 and "100k" in config["builds"]:
+        return {
+            f"metadata.tsv.xz": "results/100k/100k_subsampled_metadata.tsv.xz",
+            f"sequences.fasta.xz": "results/100k/100k_subsampled_sequences.fasta.xz"
+        }
```

Comment:
_non-blocking_

This is a bit hackish. I wonder about letting the config optionally specify the set of files to upload instead.

Reply:
For your consideration: https://github.com/nextstrain/ncov/commit/595b96367d928d08945bd7ce3fc2f0d1bbe911e3

Reply:
Thanks! Incorporated in [a41a9f6](https://github.com/nextstrain/ncov/pull/1032/commits/a41a9f63e4f1ad7c05e1d176ecfc37d38817e5c5)

I changed `config["upload"]` to `config["upload_targets"]` because a bunch of our workflows already defined that key, although I don't think the workflow actually uses it 🤷 . For instance:

https://github.com/nextstrain/ncov/blob/0a093c0677a1ac9c64564572bd43cb7bf8602eda/nextstrain_profiles/nextstrain-gisaid/builds.yaml#L13-L14

Reply:
Hmm. If we don't use `upload`, then I'd be inclined to take it over for this use. `upload_targets` is not as clear (we don't use "target" very much at all in this workflow) or as consistent with other rule-based config (e.g. `filter`).

Reply:
Sure. Done in [6b7c9e9](https://github.com/nextstrain/ncov/pull/1032/commits/6b7c9e9424e620a3ca1ae3a3085b91e8e6f7c331).

Reply:
Ah, I also just did https://github.com/nextstrain/ncov/commit/0a04128b5d7274ab22f8be64a6f2e859fadbdf46.

Reply:
All good. I'll rebase & merge tomorrow.